clang-tools-extra/clangd/AST.h

```diff
@@ -253,6 +254,23 @@ resolveForwardingParameters(const FunctionDecl *D, unsigned MaxDepth = 10);
 /// reference to one (e.g. `Args&...` or `Args&&...`).
 bool isExpandedFromParameterPack(const ParmVarDecl *D);
 
+/// Heuristic that checks if FT is forwarding a parameter pack to another
```

Comment:
"if FT is likely to be forwarding ..."

(we basically guess based on the signature only, we don't check if it actually does forwarding)